### DISCUSSION TITLE
Flow calibration prints unusably slow

### DISCUSSION THREAD

**Discussion by manD0rk** [supermerill/SuperSlicer#1395] [Q&A]
I started the usual flow calibration for some different filament. However, after the purge line and the skirt, and the very first perimeter of the first object; my Ender 3 Max suddenly slows down it's printing speed down to what looks like 1 mm/s at most. Very strange. I was trying to post my configuration file, but failed to find it on my Mac. You'll probably need that to help me, so I'll post it as soon as I know how (Google was not helpful here for me).

**Comment by supermerill** [comment]
just save your project, it contains the current configuraiton
also it's a problem/issue, so you should open an issue instead of a discussion.

**Reply by manD0rk** [reply]
Sorry man, long day but I should know better lol. I'll move this on over to the right area tomorrow. Thanks for this project btw. I've used nothing but Superslicer since I was told about it on the Voron discord a month or so back.
